### DISCUSSION TITLE
Speeding up the training rate

### DISCUSSION THREAD

**Discussion by Navdeepuniyal** [stanfordmlgroup/ngboost#249] [Q&A]
I am training a big dataset. the hyperparameter optimization resulted in 1500 estimators with a learning rate of 0.2. Now the algorithm trains quite slow as I am using walk forward validation. Is there a way to increase the training speed (something like we have in XGBoost using n_jobs )
